hack/check-files-alignment.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash -e
+
+#
+# Copyright 2019 IBM Corp.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+source hack/project_info.sh
+roles_yaml_path=config/rbac/role.yaml
+crd_yaml_path=config/crd/bases/csi.ibm.com_ibmblockcsis.yaml
+
+check_generation (){
+  echo "check generation"
+  project_dirname=ibm-block-csi-operator
+  cd ..
+  cp -r $project_dirname ./$project_dirname-expected
+  cd $project_dirname-expected/
+  make update
+  cd ..
+  diff -qr --exclude=bin $project_dirname $project_dirname-expected/
+  rm -rf $project_dirname-expected/
+  cd $project_dirname
+}
+
+verify_yaml_doc_in_multidoc_yaml(){
+  yamls_dictionary=${1#*=}
+  eval "declare -A yaml_kinds_by_origin_yaml_path="${yamls_dictionary}
```

Comment:
https://stackoverflow.com/a/3467959
>Avoid eval like the plague

Reply:
I tried without it and I didn't manage to find a solution, the variable pass between the functions interrupted it, you sent an example without a function

Reply:
I [don't think we need a dictionary](https://github.com/IBM/ibm-block-csi-operator/pull/226#discussion_r774186988)